devops/templates/pypi-deployment-stages-template.yml

```diff
@@ -51,37 +51,38 @@ stages:
     pool:
       vmImage: ${{parameters.poolImage}}
 
-    variables:
-      packageDirectory: '${{parameters.packageArtifactStem}}${{parameters.targetType}}'
-      ${{ if eq(parameters.targetType, 'Test') }}:
-        pypiUrl: https://test.pypi.org/legacy/
-      ${{ if eq(parameters.targetType, 'Prod') }}:
-        pypiUrl: https://upload.pypi.org/legacy/
-
     strategy:
-      runOnce:
-       deploy:
-        steps:
-          - task: UsePythonVersion@0
-            displayName: 'Use Python ${{parameters.poolPythonVersion}}'
-            inputs:
-              versionSpec: ${{parameters.poolPythonVersion}}
-              addToPath: true
-
-          - task: AzureKeyVault@1
-            displayName: "Fetch deployment secrets from KeyVault"
-            inputs:
-              azureSubscription: ${{parameters.kvSubscription}}
-              keyVaultName:  ${{parameters.kvVaultName}}
-              secretsFilter: '${{parameters.kvUsername}},${{parameters.kvPassword}}'
-
-          - script: pip install twine
-            displayName: 'Install twine'
-
-          # Quote the password argument in case it contains characters the shell might try to parse
-          - script: 'twine upload --verbose --repository-url $(pypiUrl) -u $(${{parameters.kvUsername}}) -p "$(${{parameters.kvPassword}})" $(Pipeline.Workspace)/$(PackageDirectory)/*'
-            displayName: "Run twine for upload"
-    
+      matrix:
```

Comment:
I hadn't spotted this before. Having this in here.... we should definitely do a 0.4.2 release to make sure everything still works